### ISSUE TITLE
There are still gems with invalid date specs on rubygems.org

### ISSUE THREAD

**Issue by sodabrew** [rubygems/rubygems.org#495]
Hi, I'm having a problem with `term-ansicolor` and other gems uploaded late last year with invalid time specifications. I am installing onto Ubuntu 12.04 LTS with Ruby 1.8.7 and RubyGems 1.8.15. These are versions that need to be reasonably well supported for a while to come, and yet...

``` shell
vendor/environment.rb:45: /usr/lib/ruby/vendor_ruby/1.8/rubygems/specification.rb:1000:in `date=': invalid date format in specification: "2011-10-13 00:00:00.00000000Z" (Gem::InvalidSpecificationException)
```

So this looks a lot like rubygems pull 57, and countless related issues. https://github.com/rubygems/rubygems/pull/57

Ok, what's going on here? **Why is this bug still affecting me, today?** Is the problem now that gems uploaded to rubygems.org during the reign of the busted rubygems versions need to be rebuilt and uploaded again?

Are there client side workarounds that _do not require newer rubygems_ on my production boxes? (Because rubygems 1.8.15 is what ships and is maintained by Ubuntu, my prod boxes will not change unless you convince Ubuntu to update rubygems in their precise-updates repo, and my guess is that ain't happening.)

Could rubygems.org could run discovery and flag the offending gems?
Basically, for every gem:

``` shell
for each gem $foo:
  tar -Oxf $foo.gem metadata.gz  | gunzip -dc - | grep date: | grep 00.000000000; if [ $? -eq 0 ]; then echo "Busted gem $foo"; fi
```
